package.json

```diff
@@ -163,7 +163,7 @@
     "karma-viewport": "^1.0.7",
     "lodash": "^4.17.20",
     "madge": "^3.12.0",
-    "mathjax": "2.7.9",
+    "mathjax": "2.7.5",
```

Comment:
The reason why we stick to `2.7.5` for now.
@nicolaskruchten commented on slack: 
```
Let's target 2.7.5 for MathJax for now
this is what JupyterLab 2.x uses
so it's a good floor IMO
JupyterLab 3.x is on 2.7.9
```